Watchman.DomainModel/Mute/MuteEvent.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Watchman.Common.Models;
+using Watchman.Integrations.MongoDB;
+
+namespace Watchman.DomainModel.Mute
```

Comment:
jeśli to ma być event, to lepiej wrzucać w namespace .Events 

w przyszłości doda sie obsługe eventów, tak jak robi sie to z komendami albo zapytaniami (queries)